lnwallet/channel.go

```diff
@@ -3572,6 +3579,13 @@ func ChanSyncMsg(c *channeldb.OpenChannel) (*lnwire.ChannelReestablish, error) {
 		return nil, err
 	}
 
+	// If we've restored this channel, then we'll purposefully give them an
```

Comment:
We need this if we've lost state before we've actually done any updates. Otherwise, everything checks out to the remote party, and they don't force close at all. In any case the remote party should force close as we sent it funky data. We already do this, and I think it's also in the spec to force close if you can't verify their commitment secret.